MvvmCross/ViewModels/MvxAppStart.cs

```diff
@@ -28,35 +28,60 @@ public void Start(object hint = null)
 
         public virtual void ResetStart()
         {
+            Reset();
             Interlocked.Exchange(ref startHasCommenced, 0);
         }
+
+        protected virtual void Reset()
```

Comment:
Should this be empty?

Reply:
no longer empty in base mvxappstart class